ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,205 @@
+
+4. If desired, click a process to display the *Process ID* view, which includes a diagram of the process and detailed information about the individual process actions and tokens.    
+For detailed information about individual process actions, see [Check the process action status](#check-the-process-action-status) .   
+For detailed information about tokens, see [Check the token status and content](#check-the-token-status-and-content).
+
+### Next steps
+
+- [Check the process action status](#check-the-process-action-status)
+- [Check the token status and content](#check-the-token-status-and-content)
+
+### See also
+
+- [Retry a single process action](/ActindoWorkFlow/Troubleshooting/02_RetryProcessAction.md#retry-a-single-process-action)
+- [Retry multiple process actions](/ActindoWorkFlow/Troubleshooting/02_RetryProcessAction.md#retry-multiple-process-actions)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Workflows](/ActindoWorkFlow/UserInterface/02a_Workflows.md)
+- [Processes](/ActindoWorkFlow/UserInterface/03a_Processes.md)
+- [Process actions](/ActindoWorkFlow/UserInterface/04a_ProcessActions.md)
+
+
+## Check the process action status
+
+The process action status can be checked to monitor which actions have already been executed, if problems have occurred and at what place.
+
+The *Process ID* view displays a diagram of the selected process including all places and actions. If the process is currently active, the process execution can be observed in real time.
+
+Further details about the process actions included in the diagram as well as about the logs and tokens are displayed in separate tabs at the bottom of the *Process ID* view.
+
+### Prerequisites
+
+- A workflow is created, see [Create a workflow](01_ManageWorkflows.md#create-a-workflow).
+- A workflow process has been initiated.
+
+### Procedure
+
+*Workflows > Processes > Tab OVERVIEW > Select process*
+
+![Process actions](/Assets/Screenshots/ActindoWorkFlow/Workflows/ProcessActions.png "[Process actions]")
+
+1. Click the *Actions* tab at the bottom of the *Process ID* view.   
+  The *Actions* tab is displayed.
+
+  > [Info] By default, the *Actions* tab is preselected.
+
+2. Click the *Status* drop-down list and select the **All** option.    
+  All actions, regardless of their status, are displayed.
+
+
+[comment]: <> (OLI: wenn eine Action/Token in einem Prozess ausgewählt werden, wird es als Filtereinstellung übernommen. Diese Filtereinstellung bleibt erhalten, auch man zu einem anderen Prozess wechlselt, daher sieht man nicht alle Actions, auch wenn auf "All" klickt. Bug oder erwartetes Verhalten?)
+
+3. Check the actions displayed in the diagram. The workflow process diagram displays the actions in different colours depending on their current status:
+
+  - Active (blue)
+  - Done (green)
+  - Error (red)
+  - Error; automatic retry (red?)
```

Comment:
does not have a color, because that state is never displayed, it is automatically back on "active" since its retried immediately